avocado/utils/cpu.py

```diff
@@ -97,28 +97,20 @@ def get_cpu_arch():
     """
     Work out which CPU architecture we're running on
     """
+    cpu_table = [('^cpu.*(RS64|POWER3|Broadband Engine)', 'power'),
+                 ('^cpu.*POWER4', 'power4'),
+                 ('^cpu.*POWER5', 'power5'),
+                 ('^cpu.*POWER6', 'power6'),
+                 ('^cpu.*POWER7', 'power7'),
+                 ('^cpu.*POWER8', 'power8'),
+                 ('^cpu.*PPC970', 'power970'),
```

Comment:
Well I didn't explicitly NACK it, I just pointed out that the whole function is quite outdated and I suggested some pointers, expecting discussion, rather than new version with all of the fixes. Obviously ideally I'd like you to take a look at it and send a v2 with all the fixes, but simple comment I don't have time for this at this moment would be also acceptable (given that new trello card is created describing the issues pointed here).

Anyway, please accept my apology if I wrote it in too strict way, I usually expect discussion before new version is send and IMO so far it worked well for me (don't know whether for others as well, but I don't know of other way).

Now back to the topic, thank you for the card, I already saw it. With those two issues taken care of I'll double check it works well and I can change the result of the review.

PS: If the hurt feeling were caused by "requested changes" I can start using "comment" only, but currently I use it to see whether I finished the review or not. I can always "Approve changes" not only when there are commit changes, but also when the situation clears out like with your reactions.